blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricTokensHandler.java

```diff
@@ -40,10 +40,10 @@ public HttpMetricTokensHandler(DiscoveryIO discoveryHandle) {
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
 
-        final Timer.Context httpMetricNameTokensHandlerTimerContext = HttpMetricNameTokensHandlerTimer.time();
-
         Tracker.getInstance().track(request);
 
```

Comment:
note:  this was moved because I don't think the timer metrics for this handler should be affect by the tracking call if it's turned on.